libGitWrap/Commit.cpp

```diff
@@ -347,23 +345,23 @@ namespace Git
 
     DiffList Commit::diffFromParent(Result& result, unsigned int index)
```

Comment:
It is named `diffFromParent` because it does a diff _from_ the parent and not _to_ the parent.